src/Contracts/Parameters/CombinedCheckerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Neomerx\JsonApi\Contracts\Parameters;
+
+/**
+ * Interface CombinedCheckerInterface
+ * @package Neomerx\JsonApi
+ */
+interface CombinedCheckerInterface extends HeaderCheckerInterface, ParameterCheckerInterface
```

Comment:
Wouldn't it be more reasonable to think that ParametersChecker is (HeadersChecker and QueryChecker)? Yes, it will break compatibility but I was thinking moving to v0.6 from v0.5.13 anyways. Feel free to rename with those or better names.